src/lib/store.js

```diff
@@ -402,9 +400,6 @@ export default class Store {
 
 	toggleConsentToolShowing = (isShown) => {
 		this.isConsentToolShowing = typeof isShown === 'boolean' ? isShown : !this.isConsentToolShowing;
-		if (this.isConsentToolShowing) {
-			this.cmpApi.update(this.persistedConsentString, true);
-		}
```

Comment:
It seems to me that by simply removing this code we won't handle case when user manually triggers consent tool showing.

Reply:
done